tests/test_transforms/test_detrend_transform.py

```diff
@@ -189,3 +196,35 @@ def test_inverse_transform_theil_sen_trend_two_segments(df_two_segments: pd.Data
     """
     trend_transform = TheilSenTrendTransform(in_column="target", n_subsamples=len(df_two_segments))
     _test_inverse_transform_many_segments(trend_transform=trend_transform, df=df_two_segments)
+
+
+def test_fit_transform_linear_trend_two_segments_diff_size(df_two_segments_diff_size: pd.DataFrame):
```

Comment:
For repeated test cases it's better to use https://docs.pytest.org/en/6.2.x/parametrize.html, you could pass transform as parameter to test case.

Reply:
Done